### DISCUSSION TITLE
Duration when starting climate control

### DISCUSSION THREAD

**Discussion by Aber-s** [Hyundai-Kia-Connect/kia_uvo#273] [Q&A]
First of all; great thanks to all of you for making this integration, it works really well and the options are amazing. I'm using it more often then the native app. I got a question about the service start climate control. I got the possibility to set up a duration. What does this duration do? I couldn't find it in the readme.

Thanks in advance and keep up the good work!

**Comment by cdnninja** [comment]
Thanks for the support! 

It depends on the region.   In most cases it is how many minutes to run the engine.   I am not sure that applies on an EV though.

**Reply by Aber-s** [reply]
Forgot to mention, it's about an IONIQ 5 so full EV. Hadn't had the chance to change the number for the duration yet.
